src/components/LoansByCategory/QuickFilters/CheckboxList.vue

```diff
@@ -78,7 +78,11 @@ export default {
 					if (exists) this.selected.splice(index, 1);
 				} else if (!exists) this.selected.push(item.value);
 			});
-			this.updateSelected(this.selected, undefined, true);
+			if (isAll && typeof item === 'object') {
```

Comment:
`item` wouldn't be defined here at all right?

Reply:
Logic updated, thanks!